rootfs/usr/local/include/toolbox/kops/Makefile.defaults

```diff
@@ -9,7 +9,7 @@ export KOPS_NAME ?= $(CLUSTER_NAME)
 export KOPS_STATE_STORE ?= s3://$(CLUSTER_STATE_BUCKET)
 
 export KOPS_CLOUD ?= aws
-export KOPS_ZONES ?= us-west-2a
+export KOPS_ZONES ?= $(AWS_REGION)a
```

Comment:
This might not always work, since zones are not the same for all AWS accounts, but I don't have a better solution right now.